website/docs/sql-function-glossary/sql-aggregate-functions/sql-count.md

```diff
@@ -0,0 +1,61 @@
+---
+id: sql-count
+title: SQL COUNT
+description: COUNT is an aggregate function that is used to return the count of rows of a specified field or all rows in a dataset. It is commonly used to get baseline statistical information of a dataset, help ensure primary keys are unique, and calculate business metrics.
+slug: /sql-function-glossary/sql-count
+---
+
+<head>
+    <title>Working with the SQL COUNT function</title>
+</head>
+
+COUNT is a SQL function you need to know how to use. Whether it’s in an ad hoc query, a data model, or in a BI tool calculation, you’ll be using the SQL COUNT function countless times (pun intended) in your data work.
+
+To formalize it, COUNT is an aggregate function that is used to return the count of rows of a specified field (`count(<field_name>)`) or all rows in a dataset (`count(*)`). It is commonly used to get baseline statistical information of a dataset, help ensure primary keys are unique, and calculate business metrics.
+
+## How to use SQL COUNT in a query
+
+Use the following syntax to generate the aggregate count of a field:
+
+`count(<field_name>)`
+
+Since COUNT is an aggregate function, you’ll need a GROUP BY statement in your query if you’re looking at counts broken out by dimension(s). If you’re calculating the standalone counts of fields without the need to break them down by another field, you don’t need a GROUP BY statement.
+
+Let’s take a look at a practical example using COUNT, DISTINCT, and GROUP BY below.
+
+### COUNT example
+
+```sql
+select
+	date_part('month', order_date) as order_month,
+	count(order_id) as cnt_all_orders,
```

Comment:
Should we just spell out `count_all_orders` for clarity?

Reply:
Sure that works for me